frost-core/src/lib.rs

```diff
@@ -173,28 +173,28 @@ pub trait Ciphersuite: Copy + Clone + PartialEq {
     ///
     /// Maps arbitrary inputs to non-zero `Self::Scalar` elements of the prime-order group scalar field.
     ///
-    /// [H1]: https://github.com/cfrg/draft-irtf-cfrg-frost/blob/master/draft-irtf-cfrg-frost.md#cryptographic-hash
+    /// [H1]: https://www.ietf.org/archive/id/draft-irtf-cfrg-frost-10.html#name-cryptographic-hash-function
```

Comment:
Note to selves, we need to review these links each time we bump the version const